components/modal/demo/info.md

```diff
@@ -1,20 +1,28 @@
 ---
 order: 5
-title: 信息提示
+title: 
+  zh-CN: 信息提示
+  en-US: Infomation modal dialog
 ---
 
+## zh-CN
+
 各种类型的信息提示，只提供一个按钮用于关闭。
 
+## en-US
+
+Various types of informaion modal dialog, only provide one button to close dialog.
```

Comment:
In the various types of information modal dialog, only one button to close the dialog is provided.